### DISCUSSION TITLE
"REMOTE MANAGMENT" remove - MacBook A1502 bios dumb

### DISCUSSION THREAD

**Discussion by MontyP2024** [MuertoGB/MacEfiToolkit#18] [General]
When reinstalling the system in my Macbook pro A1502, I get the "Remote Management" information. If I read the dump from the eeprom memory and unlocked it with your program, would this limitation (remote management) also be removed?

**Comment by MuertoGB** [comment]
No, remote management, or MDM, is hooked to the system serial number and is stored on an Apple server. If it's your device, you'll need to remove it from your MDM configuration server.

**Reply by MontyP2024** [reply]
I see. Thank you for your message
